core/src/test/java/org/elasticsearch/search/sort/FieldSortBuilderTests.java

```diff
@@ -0,0 +1,90 @@
+/*
+x * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.search.sort;
+
+import java.io.IOException;
+
+public class FieldSortBuilderTests extends AbstractSearchSourceItemTestCase<FieldSortBuilder> {
+
+    @SuppressWarnings("unchecked")
+    public Class<FieldSortBuilder> getPrototype() {
```

Comment:
I find this method name missleading, since it doesn't return the prototype builder but its class. Also see above comment, I think we should register all NamedWritable implementations that belong together under one category like we did for queries (QueryBuilder.class, here probably SortBuilder.class)